pkg/server/plugin/upstreamauthority/awspca/pca_test.go

```diff
@@ -300,93 +320,79 @@ func (as *PCAPluginSuite) Test_MintX509CA_GetCertificate_CertificateChainParseEr
 	_, encodedCert := as.SVIDFixture()
 	as.verifyGetCertificate(encodedCert, nil, nil)
 
-	// The resulting response should error
-	response, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
+	stream, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
 		Csr:          csr,
 		PreferredTtl: testTTL,
 	})
+	as.Require().NoError(err)
+	as.Require().NotNil(stream)
+
+	// The resulting response should error
+	response, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns `io.EOF`.